build/instrumentation.build.js

```diff
@@ -61,8 +61,21 @@ export function createConfig({
       sourcemap: true,
       assetFileNames: '[name][extname]',
       entryFileNames: 'app.js',
```

Comment:
Okey, this can't be done yet. The core chunk exports certain utilities, and wrapping it with the polyfills plugin in a closure will create invalid JS, as the import/export tokens must appear at the top level.
The only way I can imagine for this to work is to use the top-level await together with a dynamic import to the polyfills. This should ensure that the rest of the waits to be executed, but [this feature hasn't enough support yet](https://caniuse.com/mdn-javascript_operators_await_top_level).

So when we attack the polyfills task, we either inline them in the `core` chunk, or we have to leave another entry chunk like we do now.